buildSrc/src/main/java/quilt/internal/Constants.java

```diff
@@ -0,0 +1,18 @@
+package quilt.internal;
+
+public class Constants {
+    public static final String MINECRAFT_VERSION = "21w40a";
```

Comment:
Is there a way to get this from a properties file? So we don't have to change the Minecraft version in two different places (here and the buildscript)

Reply:
Im planning on some way to configure this automatically. If i can do it, there will be an update mappings task, which will change this automatically